Kitodo/src/main/java/org/goobi/io/SafeFile.java

```diff
@@ -315,4 +315,8 @@ public URI toURI() {
     public URL toURL() throws MalformedURLException {
         return delegate.toURL();
     }
+
+    public File getDelegate() {
```

Comment:
This is needed as the moved `start` method inside the `FileService` class need a `File` object instead of former `SafeFile` object. 

Maybe this method signature change should even reverted back.